<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorePanel</class>
 <widget class="QWidget" name="ScorePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>140</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Academy Engraved LET</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="my_title">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Me</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="my_score">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="points1">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="left_title">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Left Robot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="left_score">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="points2">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="right_title">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Right Robot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="right_score">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="points3">
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
